src/BroadcasttClient.php

```diff
@@ -459,7 +459,7 @@ private function post($path, $queryParams = [], $postParams = [])
      * @return ResponseInterface See Broadcastt API docs
      * @throws GuzzleException
      */
-    public function get($path, $queryParams = [])
+    public function get(string $path, $queryParams = []): ResponseInterface
```

Comment:
```suggestion
    public function get(string $path, array $queryParams = []): ResponseInterface
```

Reply:
`$queryParams` parameter can be an array or a string. This makes my suggestion invalid.